src/Microsoft.VisualStudio.Editors/ResourceEditor/xlf/OpenFileWarningDialog.zh-Hant.xlf

```diff
@@ -1,37 +1,37 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hant" original="../OpenFileWarningDialog.resx">
-    <body>
-      <trans-unit id="alwaysCheckCheckBox.Text">
-        <source>&amp;Always ask before opening this file type.</source>
-        <target state="translated">開啟這種檔案類型時永遠要詢問(&amp;A)</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="messageLabel.Text">
-        <source>The Managed Resource Editor does not support the requested file type. Would you like to open the default editor for "{0}"?</source>
-        <target state="translated">受控資源編輯器不支援要求的檔案類型。要開啟 "{0}" 的預設編輯器嗎?</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="buttonOK.Text">
-        <source>OK</source>
-        <target state="translated">確定</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="buttonCancel.Text">
-        <source>Cancel</source>
-        <target state="translated">取消</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="messageLabel2.Text">
-        <source>Some files could contain malicious content.  You should only answer Yes if you trust the source of this file.</source>
-        <target state="translated">某些檔案可能包含懷有惡意的內容。應該只在您信任此檔案的來源時，才回覆 [是]。</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="$this.Text">
-        <source>Open File</source>
-        <target state="translated">開啟檔案</target>
-        <note />
-      </trans-unit>
-    </body>
-  </file>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
What is the actual difference in these files? I don't think these should all be displaying diffs.